### ISSUE TITLE
disk cache is not working as intended

### ISSUE THREAD

**Issue by adam-ce** [AlpineMapsOrg/renderer#7]
the caching strategies conflict. i.e., tile 0/0/0 is loaded only once, gets evicted and is then missing on the next application start. we won't display anything before it is loaded, so we wait.

**Comment by adam-ce** [comment]
in progress (branch disk_cache)

remaining tasks:

- [ ] if there is an server error, sometimes a html page is returned instead of the image. do not send that data to the gpu. send either nothing, or the default tile.
- [ ] write every x minutes to disk (after tiles received)
- [ ] loading doesn't resume with connection
- [ ] sometimes http connections are not killed, therefore load slots are not released. e.g., navigate to an area without data (white)
- [ ] the ram/disk cache is not limited to 50k (clear ram cache is not running, unit test fails)
- [ ] here is a race between the scheduler and the renderer (tiles are delivered before gl init is called. gl init is called after a certain delay in the main thread. that one takes longer than 200ms when the debugger is attached)

**Comment by adam-ce** [comment]
- [ ] do not store tiles that are 404 or otherwise broken to disk

**Comment by adam-ce** [comment]
duplicate of #22
